service-front/module/Application/src/Model/Service/Mail/Transport/MailTransportFactory.php

```diff
@@ -40,18 +39,7 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $transport = $emailConfig['transport'];
         }
 
-        if ($transport === 'sendgrid') {
-            $sendGridConfig = $emailConfig['sendgrid'];
-
-            if (!isset($sendGridConfig['key'])) {
-                throw new RuntimeException('Sendgrid settings not found');
-            }
-
-            $client = new SendGrid($sendGridConfig['key']);
-            $messageFactory = $container->get('MessageFactory');
-
-            return new SendGridMailTransport($client->client, $messageFactory);
-        } elseif ($transport === 'notify') {
+        if ($transport === 'notify') {
```

Comment:
Wouldn't we want to remove this completely? Along with removing the very end RuntimeException. As we shouldn't be dependent on config to make sure notify is specified.

Reply:
Agreed. Somehow that slipped through the net, will take that out too